manifests/s/Squitch/Tess/0.6.1/Squitch.Tess.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Squitch.Tess
+PackageVersion: 0.6.1
+PackageLocale: en-US
+Publisher: Squitch

```

```suggestion
Publisher: TessApp
```
This must match Control Panel.